e2e/rbd.go

```diff
@@ -189,12 +189,26 @@ func validateRBDImageCount(f *framework.Framework, count int, pool string) {
 		framework.Failf("failed to list rbd images: %v", err)
 	}
 	if len(imageList) != count {
+		var imageDetails []string // To collect details for all images
+		for _, image := range imageList {
+			imgInfo, err := getImageInfo(f, image, pool)
+			if err != nil {
+				return
+			}
+			imgStatus, err := getImageStatus(f, image, pool)
+			if err != nil {
+				return
```

Comment:
we need to log error and continue as this is for debugging

Reply:
done